tests/bsim/bluetooth/mesh/compile.sh

```diff
@@ -15,24 +15,16 @@ app=tests/bsim/bluetooth/mesh conf_overlay=overlay_pst.conf compile
 app=tests/bsim/bluetooth/mesh conf_overlay=overlay_gatt.conf compile
 app=tests/bsim/bluetooth/mesh conf_overlay=overlay_gatt_separate.conf compile
 app=tests/bsim/bluetooth/mesh conf_overlay=overlay_low_lat.conf compile
-app=tests/bsim/bluetooth/mesh conf_overlay=overlay_psa.conf compile
 app=tests/bsim/bluetooth/mesh conf_overlay=overlay_workq_sys.conf compile
 app=tests/bsim/bluetooth/mesh conf_overlay=overlay_multi_adv_sets.conf compile
 app=tests/bsim/bluetooth/mesh conf_overlay=overlay_lpn_scan_on.conf compile
 app=tests/bsim/bluetooth/mesh \
```

Comment:
I guess this line should be removed too. It doesn't throw error only because line 22 is empty.